pages/cloudflare/howtos/NextAuth.mdx

```diff
@@ -4,6 +4,11 @@ NextAuth.js is an open-source authentication solution for Next.js applications.
 
 ### Solving a broken build
 
+<Callout>
```

Comment:
I think you fogot to import the `Callout` component:
![Screenshot 2025-03-06 at 10 41 31](https://github.com/user-attachments/assets/bcba7de3-0128-4662-a53c-84de8c7d606c)

You can do it like so at the top of this file:
```ts
import { Callout } from "nextra/components";
```